index.html

```diff
@@ -1629,6 +1629,72 @@ <h4>DIDCommMessaging</h4>
 
       </section>
 
+      <section id="wot-discovery-service-type">
+        <h4>Web of Things (WoT) Discovery</h4>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://www.w3.org/TR/wot-discovery/">WoT Discovery</a>
+              </td>
+              <td>
+                <span class="issue">A valid JSON-LD context needs to be published.</span>
+                <a href="https://www.w3.org/2021/wot/discovery/did#WotThing">WotThing</a>
+              </td>
+            </tr><tr>
+              <td>
+                <a href="https://www.w3.org/TR/wot-discovery/">WoT Discovery</a>
+              </td>
+              <td>
+                <span class="issue">A valid JSON-LD context needs to be published.</span>
+                <a href="https://www.w3.org/2021/wot/discovery/did#WotDirectory">WotDirectory</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of WotThing serviceEndpoint properties">
+{
+  "@context":[
+      "https://www.w3.org/ns/did/v1",
+      "https://www.w3.org/2021/wot/discovery/did"
```

Comment:
https://www.w3.org/2021/wot/discovery/did is 404.... I prefer not to accept registrations that contain 404 URLs... If its blocked by W3C, maybe register a github one first?

https://w3c.github.io/wot-discovery/context/discovery-context.jsonld ?

Reply:
URL should now be resolvable, although it needs to be updated (I will ensure that it is).